src/components/CreateInput/CreateInput.tsx

```diff
@@ -0,0 +1,38 @@
+import { Dispatch, SetStateAction } from "react";
+import styles from './CreateInput.module.css';
+import {BsPlusLg} from 'react-icons/bs';
```

Comment:
Follow one code style
different indents {BsPlusLg} and { Dispatch, SetStateAction }